standardtracer/standardcontext.go

```diff
@@ -0,0 +1,48 @@
+package standardtracer
+
+import "sync"
+
+type StandardContext struct {
```

Comment:
I'm starting to doubt it :)

I can move all of this to the `dapperish` package if that's better, though I'd prefer to make that change in a separate PR regardless.